ddp-workspace/projects/ddp-pancan/src/styles/common.scss

```diff
@@ -1,40 +1,57 @@
 @import '../styles/variables.scss';
 
-.static-page-title {
+@mixin static-page-title {
     font-family: $secondary-font;
     font-size: 2.5rem;
     font-weight: 600;
     letter-spacing: -0.08px;
     line-height: 50px;
 }
 
-.static-page-paragraph-text {
+@mixin static-page-paragraph-text {
     opacity: 0.8;
     font-size: 1.5rem;
     letter-spacing: -0.1px;
     line-height: 34px;
 }
 
-.btn-common {
```

Comment:
I used btn-common, btn-outline and btn-header classes in html. I suppose it's not working now.
mixin is not always the best choice. Mixins are very good if styles are configurable based on the passed value. But for regular cases, simple classes are a better choice. It's easier to search and debug

Reply:
I was not able to find places where 
>  used btn-common, btn-outline and btn-header classes in html.

Seems like the classes are used in css only.
And they should be working currently.

Using `mixins` is just more convenient way to re-use common styles/wrap several style into one (then `@extend`), imho.
They can be used without any passed value as well. 

Anyway, would you like me to get those styles back as class selectors ?

Reply:
oh, I added the comment but didn't open the PR where I use it
https://github.com/broadinstitute/ddp-angular/pull/662/files
here

yes please! ability to use simple style classes right in html is very important, we need to keep it.

Reply:
like I said, it's easier to debug, plus you can add/remove classes conditionally in html based on some logic. We can't do it with mixins

Reply:
Reverted at 276b15b92785892acc656f8ccae03443170efcb9